### DISCUSSION TITLE
Published: A generic framework for the physiologically-based pharmacokinetic platform qualification of PK-Sim and its application to predicting cytochrome P450 3A4-mediated drug-drug interactions

### DISCUSSION THREAD

**Discussion by sfrechen** [Open-Systems-Pharmacology/Forum#929] [Announcement]
We have published the OSP framework for platform qualification with a first showcase of predicting cytochrome P450 3A4-mediated drug-drug interactions. Thanks to all contributors!

[Sebastian Frechen, Juri Solodenko, Thomas Wendl, André Dallmann, Ibrahim Ince, Thorsten Lehr  Jörg Lippert, Rolf Burghaus. **A generic framework for the physiologically-based pharmacokinetic platform qualification of PK-Sim and its application to predicting cytochrome P450 3A4-mediated drug-drug interactions**. _CPT Pharmacometrics Syst Pharmacol
. 2021 Jun;10(6):633-644_.](https://doi.org/10.1002/psp4.12636)

**Abstract**
The success of applications of physiologically-based pharmacokinetic (PBPK) modeling in drug development and drug labeling has triggered regulatory agencies to demand rigorous demonstration of the predictive capability of the specific PBPK platform for a particular intended application purpose. The effort needed to comply with such qualification requirements exceeds the costs for any individual PBPK application. Because changes or updates of a PBPK platform would require (re-)qualification, a reliable and efficient generic qualification framework is needed. We describe the development and implementation of an agile and sustainable technical framework for automatic PBPK platform (re-)qualification of PK-Sim® embedded in the open source and open science GitHub landscape of Open Systems Pharmacology. The qualification approach enables the efficient assessment of all aspects relevant to the qualification of a particular purpose and provides transparency and traceability for all stakeholders. As a showcase example for the power and versatility of the qualification framework, we present the qualification of PK-Sim® for the intended purpose of predicting cytochrome P450 3A4 (CYP3A4)-mediated drug-drug interactions (DDIs). Several perpetrator PBPK models featuring various degrees of CYP3A4 modulation and different types of mechanisms (competitive inhibition, mechanism-based inactivation, and induction) were coupled with a set of PBPK models of sensitive CYP3A4 victim drugs. Simulations were compared to a comprehensive data set of 135 observations from published clinical DDI studies. The platform's overall predictive performance showed reasonable accuracy and precision (geometric mean fold error of 1.4 for both area under the plasma concentration-time curve ratios and peak plasma concentration ratios with/without perpetrator) and suggests that PK-Sim® can be applied to quantitatively assess CYP3A4-mediated DDI in clinically untested scenarios.

**Comment by drmansoor** [comment]
I hope you are keeping well,our research team has been involved in the silico studies using pk sim since the beginning of the system.We are first who got the licence of pk sim in Pakistan. My colleague prof.Dr.Tasneem Ahmed along with my student name Sharib also joined Bayer in Germany to attend the training seminar. We also published a few articles and currently our group started a project related to pk sim. We will appreciate it if you supervise and guide in this regard.
kindly, provide me your email address for further discussion .
Thanks
Best Regards
Prof.Dr.Mansoor Ahmed.
department: pharmaceutical chemistry , university of Karachi Pakistan
